.gitlab/package_build/windows.yml

```diff
@@ -56,30 +56,39 @@
 windows_msi_and_bosh_zip_x64-a7:
```

Comment:
I thought this job was run on every branch, did this change recently?

Reply:
👍 confusion on how/when the variables are filled - they should be running on every branch.